dlio_benchmark/storage/storage_factory.py

```diff
@@ -16,7 +16,8 @@
 """
 from dlio_benchmark.storage.file_storage import FileStorage
 from dlio_benchmark.storage.s3_storage import S3Storage
-from dlio_benchmark.common.enumerations import StorageType
+from dlio_benchmark.storage.s3_torch_storage import S3PyTorchConnectorStorage
```

Comment:
move this import inside on line:32 so that when s3 is not installed the file is not called at all.